members/forms/activity_signup_form.py

```diff
@@ -69,16 +69,16 @@ def __init__(self, *args, **kwargs):
         )
 
     note = forms.CharField(
-        label="Besked til arrangør", widget=forms.Textarea, required=False
+        label="<span style='color:red'><b>Besked til arrangør</b></span> (Særlige hensyn, gener, allergi, medicin etc.)", widget=forms.Textarea, required=False
     )
     photo_permission = forms.ChoiceField(
         label="Må Coding Pirates tage og bruge billeder og videoer af dit barn på aktiviteten? (Billederne lægges typisk på vores hjemmeside og Facebook side)",
         initial="Choose",
         required=True,
         choices=(
-            ("Choose", "Vælg om vi må tage billeder"),
+            ("Choose", "Vælg om Coding Pirates må tage billeder af mit barn til denne aktivitet"),
```

Comment:
Kan vi gøre det klart at der ikke er tale om et profilbillede, medmindre vi kontakter forældrene særskilt om det?

Reply:
Dette var også diskuteret - at gøre tydeligt at det er  CP der tager billeder, og det kun er for denne aktivitet (det er ikke en generel). 
Vi skal nok se på general beskrivelse af fototilladelse (https://codingpirates.dk/fotoguide/) - er den tydelig nok, skal den korrigeres ny retspraksis etc.